src/test/contracts/recursive_test_2.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.5.0;
+
+import "embark-test-contract-0/recursive_test_3.sol";
```

Comment:
Mostly so I could test cross-package contracts, ie where `package-0` imports a contract from it's dependent `package-1`. I didn't think of OpenZeppelin had imported contracts from a different package, but could be wrong. Was thinking I could have just put the contracts in a subfolder with a `package.json` instead of creating a new repo.